src/olympia/addons/templates/addons/contributions_lightbox.html

```diff
@@ -1,5 +1,5 @@
 <div id="contribute-box" class="jqmWindow modal">
-  <form action="{{ url('addons.contribute', addon.slug) }}" method="post">
+  <form action="{{ url('addons.contribute', addon.slug) }}" method="post" data-no-csrf>
```

Comment:
I don't get it. The csrf token is right there on the line below this one, why would we set this form as not needing it when it clearly does have one already ?